numpy/core/numeric.py

```diff
@@ -6,7 +6,22 @@
 import sys
 import warnings
 
-from numpy.core import multiarray
+from . import multiarray
+from .multiarray import (CLIP, WRAP, RAISE, MAXDIMS, ALLOW_THREADS,
```

Comment:
I think this would be better in the (numpy conventional) form
```
from .multiarray import (
    CLIP, WRAP, RAISE, MAXDIMS, ALLOW_THREADS, etc
)
```